src/tools/pub/cmd/pub-outdated.md

```diff
@@ -161,13 +165,13 @@ it should be possible to update this package.
 
 ## Output columns
 
-The output of `pub outdated` has four columns of version information
+The output of `dart pub outdated` has four columns of version information
 for each out-of-date dependency.
 Here is the part of the [example](#example) output
 that shows the four version columns:
 Current, Upgradable, Resolvable, and Latest.
 
-{% include_relative _pub-outdated-output-columns.html %}
+{% include_relative _dart-pub-outdated-output-columns.html %}
```

Comment:
"_pub-outdated-ouput-columns.html" references to a file on the relative path. So I think you should not change it to "_dart-pub-outdated-ouput-columns.html" as there is no file with this name on the relative path